srcpkgs/sequoia-sq/template

```diff
@@ -1,25 +1,28 @@
 # Template file for 'sequoia-sq'
 pkgname=sequoia-sq
-version=0.29.0
+version=0.31.0
 revision=1
 build_style=cargo
-_deps="nettle-devel openssl-devel"
+build_helper=qemu
+_deps="nettle-devel openssl-devel sqlite-devel"
 hostmakedepends="pkg-config llvm clang ${_deps}"
 makedepends="${_deps}"
 short_desc="Command-line frontend for Sequoia, a new OpenPGP implementation"
 maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
 license="GPL-2.0-or-later"
 homepage="https://sequoia-pgp.org/projects/#sq"
 distfiles="https://gitlab.com/sequoia-pgp/sequoia-sq/-/archive/v${version}/sequoia-sq-v${version}.tar.gz"
-checksum=4868b45f41e04f0a0f104fd6d1fcb1c33f578c193124ed98b465167a2f6ea577
+checksum=2bb62ce5fcc33a5340dae678a6628a9be992ff234141015256deb2cf45092b44
 conflicts="squirrel"
 
 case "$XBPS_TARGET_MACHINE" in
 	armv*l) nocross="Requires C libs included in build.rs, which is currently broken in xbps-src. These failures only manifest on a hf archs right now";;
 esac
 
-pre_build() {
-	export SQ_MAN=man-sq
+post_build() {
+	SQ="target/${RUST_TARGET}/release/sq"
+	ln -sf "$SQ" sq
```

Comment:
why symlink it if you're only using it once?

Reply:
because I cargo-culted this from rustup. Fixing it now.